### PR TITLE
Fix tooltips and hotkey for day-theme

### PR DESCRIPTION
**Pull request by michaelkirk**
**Not controversial**: The tooltips were assuming the default text color, making them unreadable in the day theme (dark text on dark background), now tooltips always default to white font, while respecting any custom color (like hotkey TextSpans).

**Maybe controversial**: replace OSD "name_color" with underline

The current `name_color` isn't legible in day theme. There's not current design for this, so I went with an underline treatment instead. (If there is a preference to keep the colors, I'll revert once I get a new color from @muzlee1113)

My personal opinion is that we're doing too many different things with color, which gets even more complicated to design/implement as we've introduced day/night themes. Relying on color-neutral differentiation like text styles avoid this. Arguably, "underline" isn't ideal, since we use "underline"  elsewhere to mean "clickable hyperlink".

I actually initially tried using a Bold font instead of an underline decoration, but apparently widgetry doesn't support switching fonts mid-line.

**before**
Screen Shot 2021-02-24 at 10 15 26 AM

**after**
Screen Shot 2021-02-24 at 10 14 04 AM

### CODE REVIEW

File: fifteen_min/src/viewer.rs [hunk 1]
```diff
@@ -269,7 +269,7 @@ fn build_panel(ctx: &mut EventCtx, app: &App, start: &Building, isochrone: &Isoc
 
     rows.push(
         ctx.style()
-            .btn_light_popup_icon_text(
+            .btn_popup_icon_text(
```
Comment (michaelkirk) [inline fifteen_min/src/viewer.rs:272]: just a rename here - had a lingering "light" in the button names even though it's used in both themes.

File: game/src/common/mod.rs [hunk 2]
```diff
@@ -97,7 +97,7 @@ impl CommonState {
         } else if app.opts.dev {
             Text::from_all(vec![
                 Line("Nothing selected. Hint: "),
-                Line("Ctrl+J").fg(g.style().hotkey_color),
+                Line("Ctrl+J").fg(g.style().text_hotkey_color),
```
Comment (michaelkirk) [inline game/src/common/mod.rs:100]: another rename... trying to do a reverse polish notation thing to group the theme variables.

File: widgetry/src/drawing.rs [hunk 3]
```diff
@@ -161,7 +161,7 @@ impl<'a> GfxCtx<'a> {
     pub fn draw_mouse_tooltip(&mut self, txt: Text) {
         // Add some padding
         let pad = 5.0;
-
+        let txt = txt.default_fg(self.style.text_tooltip_color);
```
Comment (michaelkirk) [inline widgetry/src/drawing.rs:164]: This is the one place I changed to update default tooltip text color across the app

File: widgetry/src/style/mod.rs [hunk 4]
```diff
@@ -75,25 +76,54 @@ impl ButtonStyle {
 
 impl Style {
     pub fn standard() -> Style {
+        let use_legacy_day_theme = true;
```
Comment (michaelkirk) [inline widgetry/src/style/mod.rs:79]: toggling commented blocks was becoming arduous

File: map_gui/src/colors.rs [hunk 5]
```diff
@@ -191,7 +190,6 @@ impl ColorScheme {
             current_object: Color::WHITE,
             perma_selected_object: Color::BLUE,
             bottom_bar_id: Color::RED,
```
Comment (dabreegster) [inline map_gui/src/colors.rs:192]: Maybe once we can change fonts mid-line (https://github.com/a-b-street/abstreet/blob/ac615e97c3d25064ab0f9b7a335e64bdd40d02b8/widgetry/src/text.rs#L304 -- probably not too hard, and we can maybe even make usvg do more of the work with alignment), we can also get rid of the red debug IDs and make them bold (or flip the bold/underline IDs/names).

The bottom bar has never appeared in any of the Figma designs. :)  I'm solely responsible for all wacky colors there.

**Comment by dabreegster** [comment]
Almost LGTM -- the bottom bar instantly looks more polished! But the minimap buttons broke:
Screenshot from 2021-02-24 10-39-43
Screenshot from 2021-02-24 10-39-08
I don't think the light_popup rename affects this, looking into it...

> My personal opinions is that we're doing too many different things with color

Agreed. It's mostly legacy design from before Yuwen or from before text rendering was decent, that we never cleaned up.

**Comment by michaelkirk** [comment]
> I don't think the light_popup rename affects this, looking into it...

Whoops, I think I know what happened there. I'll push up a fix shortly...

File: widgetry/src/style/mod.rs [hunk 6]
```diff
@@ -75,25 +76,54 @@ impl ButtonStyle {
...
+                ButtonStyle {
+                    fg: Color::WHITE,
+                    fg_disabled: Color::WHITE.alpha(0.3),
+                    bg: hex("#4C4C4C").alpha(0.8),
+                    bg_hover: hex("#4C4C4C"),
+                    bg_disabled: Color::grey(0.6),
+                    outline: hex("#4C4C4C").alpha(0.6),
+                }
+            },
+            btn_solid_floating: if use_legacy_day_theme {
```
Comment (dabreegster) [inline widgetry/src/style/mod.rs:122]: Aha, these two cases are inverted. See the minimap

Comment (michaelkirk) [reply widgetry/src/style/mod.rs:122]: Thank you! Fixed

**Pull request merged**